skyportal/plot.py

```diff
@@ -145,6 +145,7 @@ def _plot_to_json(plot):
     return docs_json, render_items, custom_model_js
 
 
+# TODO async
```

Comment:
What was our decision here?

Reply:
Not related to this PR, just a reminder. I clarified the TODO text a bit.